internal/email/email.go

```diff
@@ -61,56 +108,84 @@ func NewMessage(
 	return message.NewMessage(id.String(), payload), nil
 }
 
-// getBodyHTML returns the HTML body for the email based on the message payload
-func getEmailBodyHTML(ctx context.Context, inviteURL, minderURL, sponsor, project, role, inviteeEmail string) string {
-	data := struct {
-		AdminName        string
-		OrganizationName string
-		InvitationURL    string
-		RecipientEmail   string
-		MinderURL        string
-		TermsURL         string
-		PrivacyURL       string
-		SignInURL        string
-		RoleName         string
-		RoleVerb         string
-	}{
-		AdminName:        sponsor,
-		OrganizationName: project,
-		InvitationURL:    inviteURL,
-		RecipientEmail:   inviteeEmail,
-		MinderURL:        minderURL,
-		TermsURL:         DefaultMinderTermsURL,
-		PrivacyURL:       DefaultMinderPrivacyURL,
-		SignInURL:        minderURL,
-		RoleName:         role,
-		RoleVerb:         authz.AllRolesVerbs[authz.Role(role)],
-	}
-
-	// TODO: Load the email template from elsewhere
+// getHTMLBodyString returns the HTML body for the email based on the message payload.
+// If there is an error creating the HTML body, it will try to return the text body instead
+func getEmailBodyHTML(ctx context.Context, data bodyData) (string, error) {
+	var b strings.Builder
+	bHTML := bodyHTML
 
-	// Parse the template
-	tmpl, err := templates.Parse(bodyHTML)
+	bodyHTMLTmpl, err := util.NewSafeHTMLTemplate(&bHTML, "body-invite-html")
```

Comment:
Don't fix this now, but we should be able to load this template _once_ and then invoke it a number of times, rather than loading it every single time.  (And then, if we get a nice data push set up, we'll need to be able to load it dynamically, but right now this is either being put there by an init container, or being put there by the container install.)

Also, rather than using `bHTML`, you should be able to put the file in `/cmd/server/kodata/templates/email.tmpl` and then read it from `$KO_DATA_PATH/templates/email.tmpl`.  I'd hold off on that for a different PR, though.